src/libraries/System.ObjectModel/tests/TrimmingTests/TypeDescriptionProviderAttributeCtor.cs

```diff
@@ -0,0 +1,16 @@
+using System;
```

Comment:
This test maps to https://github.com/dotnet/runtime/pull/37402/files#diff-63a0199bf72757c4ad54a78ce257e9dcR28.